cogboard-webapp/src/components/widgets/types/WorldClockWidget.js

```diff
@@ -0,0 +1,27 @@
+import React, {useState} from 'react';
+import {useInterval} from '../../../hooks'
+import moment from 'moment-timezone'
+import Typography from "@material-ui/core/Typography";
+
+const WorldClockWidget = props => {
+    const {timeZoneId, dateFormat} = props;
+
+    const getDate = (timezone, format) => {
```

Comment:
Move it to component helpers. Make a folder for component and index.js and helpers.js. Look at WidgetTypeForm for the reference.